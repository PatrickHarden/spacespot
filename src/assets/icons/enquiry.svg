<?xml version="1.0" encoding="UTF-8"?>
<svg width="30px" height="28px" viewBox="0 0 30 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: sketchtool 63 (101010) - https://sketch.com -->
    <title>F65E5ADF-220D-4D3F-9779-B9713793BF78</title>
    <desc>Created with sketchtool.</desc>
    <g id="Header" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="Full-home-page-expanded-Copy-3" transform="translate(-292.000000, -748.000000)" fill="#FFFFFF" fill-rule="nonzero">
            <g id="enquiry-icon" transform="translate(292.000000, 748.000000)">
                <path d="M26.4983523,0.262180398 L9.76683239,0.22787642 C8.16434659,0.22787642 6.86079545,1.53142756 6.86079545,3.13391335 L6.86079545,5.64496449 L2.90603693,5.65329545 C1.30355114,5.65329545 0,6.95684659 0,8.55933239 L0,18.9196236 C0,20.5221094 1.30355114,21.8251705 2.90603693,21.8251705 L5.88068182,21.8251705 L5.88068182,26.7257386 C5.88068182,26.9281321 6.00515625,27.1099432 6.19431818,27.1829616 C6.25165483,27.2050142 6.31144176,27.2157955 6.37073864,27.2157955 C6.50599432,27.2157955 6.63879972,27.1594389 6.73338068,27.0555469 L11.488892,21.8241903 L19.6365767,21.7908665 C21.2390625,21.7908665 22.5426136,20.4873153 22.5426136,18.8853196 L22.5426136,18.8299432 L25.1203125,21.6654119 C25.2148935,21.769304 25.3476989,21.8256605 25.4829545,21.8256605 C25.5422514,21.8256605 25.6020384,21.8148793 25.659375,21.7928267 C25.8485369,21.7198082 25.9730114,21.5379972 25.9730114,21.3356037 L25.9730114,16.4350355 L26.4973722,16.4350355 C28.099858,16.4350355 29.4034091,15.1314844 29.4034091,13.5294886 L29.4034091,3.16821733 C29.4034091,1.56622159 28.100348,0.262670455 26.4983523,0.262180398 Z M21.5625,18.8848295 C21.5625,19.9467827 20.6985298,20.8102628 19.6346165,20.8102628 L11.2693466,20.8445668 C11.1321307,20.8450568 11.0007955,20.9033736 10.9086648,21.0048153 L6.86079545,25.4579616 L6.86079545,21.3346236 C6.86079545,21.0641122 6.64174006,20.8445668 6.37073864,20.8445668 L2.90603693,20.8445668 C1.84408381,20.8445668 0.980113636,19.9805966 0.980113636,18.9191335 L0.980113636,8.55884233 C0.980113636,7.4968892 1.84408381,6.63291903 2.90701705,6.63291903 L7.35085227,6.62360795 C7.35085227,6.62360795 7.35085227,6.62360795 7.35085227,6.62360795 C7.35134233,6.62360795 7.35134233,6.62360795 7.35183239,6.62360795 L19.6365767,6.598125 C20.6985298,6.598125 21.5625,7.46209517 21.5625,8.5240483 L21.5625,17.5611861 L21.5625,18.8848295 Z M28.4232955,13.5285085 C28.4232955,14.5904616 27.5593253,15.4539418 26.4973722,15.4539418 L25.4829545,15.4539418 C25.2119531,15.4539418 24.9928977,15.6734872 24.9928977,15.9439986 L24.9928977,20.0673366 L22.5426136,17.3720241 L22.5426136,8.52453835 C22.5426136,6.92205256 21.2390625,5.61850142 19.6355966,5.61850142 L7.84090909,5.6425142 L7.84090909,3.1334233 C7.84090909,2.07147017 8.70487926,1.2075 9.76585227,1.2075 L26.4968821,1.24180398 C26.4973722,1.24180398 26.4973722,1.24180398 26.4978622,1.24180398 C27.5598153,1.24180398 28.4237855,2.10577415 28.4237855,3.16772727 L28.4237855,13.5285085 L28.4232955,13.5285085 Z" id="Shape"></path>
            </g>
        </g>
    </g>
</svg>