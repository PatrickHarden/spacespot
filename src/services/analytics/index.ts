import ReactGA from 'react-ga'
import axios from 'axios'

import config from 'config'
import history from 'browserhistory'
import { loadPixelAPI } from './loadpixel'
export { usePixel } from './hooks'

let ready = false

const init = () => {
  const consent = localStorage.getItem('cookie-consent')
  if (!consent) return
  ReactGA.initialize(config.GOOGLE_TRACKING_CODE)
  ready = true
  // console.log('Analytics: init')

  // Initialize google analytics page view tracking
  history.listen(location => {
    // console.log('Analytics: pageView ', location.pathname)
    ReactGA.set({ page: location.pathname }) // Update the user's current page
    ReactGA.pageview(location.pathname) // Record a pageview for the given page
  })

  if (consent !== 'false') {
    loadPixelAPI()
  }
}

const event = (ev: any) => {
  if (!ready) return
  // console.log('Analytics: event', ev)
  ReactGA.event(ev)
}

/**
 * GTM dataLayer push
 *
 * see https://developers.google.com/tag-manager/devguide#events
 */
const gtmEvent = (ev: any) => {
  const win = (window as unknown) as any
  win.dataLayer && win.dataLayer.push(ev)
}

const pixel = (ct: string) => {
  const consent = localStorage.getItem('cookie-consent')

  if (!ready) return
  try {
    if (consent !== 'false') {
      axios.get(
        // Aded withCredentials to forward the token for Campaign
        //`https://insight.adsrvr.org/track/pxl/?adv=6gmmyim&ct=${ct}&fmt=3`,
        `https://insight.adsrvr.org/track/pxl/?adv=6gmmyim&ct=${ct}&fmt=3`, { withCredentials: true },
      )
    }
  } catch {
    // ignore pixel issues
  }
}

const PIXELS = {
  SEARCH: '0:ka4bzg6',
  SELECT_SPACE: '0:btb7pw8',
  SEND_ENQUIRY: '0:6ywcv06',
  SIGNUP: '0:6u9gs4z',
  ACCEPT_ENQUIRY: '0:9jr4h2a',
  NEGOTIATION_SUBMITTED: '0:pcdfo63',
  ACCEPT_LEASE: '0:2tge50w',
  SIGN: '0:bqnev20',
}

export default {
  init,
  event,
  gtmEvent,
  pixel,
  PIXELS,
}
