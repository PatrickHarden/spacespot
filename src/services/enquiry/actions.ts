import {
  EnquiryAction,
  EnquiryResponse,
  DocumentDTO,
  DocData,
  EnquiryRequest,
  EnquiryRequestFlex,
} from './types'
import { FileItem } from 'services/global/types'
import constants from './constants'

const init = (payload: EnquiryRequest | EnquiryRequestFlex) => ({
  type: constants.INIT,
  payload,
})

const initSuccess = (payload: EnquiryResponse): EnquiryAction => ({
  type: constants.INIT_SUCCESS,
  payload,
})

const initError = (payload: string): EnquiryAction => ({
  type: constants.INIT_ERROR,
  payload,
})

const get = (payload: string): EnquiryAction => ({
  type: constants.GET,
  payload,
})

const getSuccess = (payload: EnquiryResponse): EnquiryAction => ({
  type: constants.GET_SUCCESS,
  payload,
})

const getError = (payload: string): EnquiryAction => ({
  type: constants.GET_ERROR,
  payload,
})

const getDocs = (payload: number): EnquiryAction => ({
  type: constants.GET_DOCS,
  payload,
})

const getDocsSuccess = (payload: Array<DocumentDTO>): EnquiryAction => ({
  type: constants.GET_DOCS_SUCCESS,
  payload,
})

const getDocsError = (payload: string): EnquiryAction => ({
  type: constants.GET_DOCS_ERROR,
  payload,
})
const updateDocs = (payload: number): EnquiryAction => ({
  type: constants.UPDATE_DOCS,
  payload,
})

const putDocType = (item: FileItem, docType: string): EnquiryAction => ({
  type: constants.PUT_DOC_TYPE,
  payload: {
    item: item,
    docType: docType,
  },
})

const putDocTypeSuccess = (payload: DocumentDTO): EnquiryAction => ({
  type: constants.PUT_DOC_TYPE_SUCCESS,
  payload,
})

const putDoc = (payload: FileItem): EnquiryAction => ({
  type: constants.PUT_DOC,
  payload,
})

const putDocSuccess = (payload: DocumentDTO): EnquiryAction => ({
  type: constants.PUT_DOC_SUCCESS,
  payload,
})

const putDocError = (payload: string): EnquiryAction => ({
  type: constants.PUT_DOC_ERROR,
  payload,
})

const getDoc = (payload: DocumentDTO): EnquiryAction => ({
  type: constants.GET_DOC,
  payload,
})

const getStdTemplate = (): EnquiryAction => ({
  type: constants.GET_STD_TEMPLATE,
  payload: null,
})

const getDocSuccess = (payload: DocData): EnquiryAction => ({
  type: constants.GET_DOC_SUCCESS,
  payload,
})

const getDocError = (payload: string): EnquiryAction => ({
  type: constants.GET_DOC_ERROR,
  payload,
})

const deleteDoc = (payload: number): EnquiryAction => ({
  type: constants.DELETE_DOC,
  payload,
})

const deleteDocSuccess = (payload: number): EnquiryAction => ({
  type: constants.DELETE_DOC_SUCCESS,
  payload,
})

const deleteDocError = (payload: string): EnquiryAction => ({
  type: constants.DELETE_DOC_ERROR,
  payload,
})

const resetNotifications = (payload: number): EnquiryAction => ({
  type: constants.RESET_NOTIFICATIONS,
  payload,
})

export default {
  init,
  initSuccess,
  initError,
  get,
  getSuccess,
  getError,
  getDocs,
  getDocsSuccess,
  getDocsError,
  updateDocs,
  putDoc,
  putDocSuccess,
  putDocType,
  putDocTypeSuccess,
  putDocError,
  getDoc,
  getStdTemplate,
  getDocSuccess,
  getDocError,
  deleteDoc,
  deleteDocSuccess,
  deleteDocError,
  resetNotifications,
}
