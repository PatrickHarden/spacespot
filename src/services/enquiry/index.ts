import actions from './actions'
import constants from './constants'
import reducer from './reducer'
import saga from './saga'

export * from './types'
export { actions, constants, reducer, saga }
