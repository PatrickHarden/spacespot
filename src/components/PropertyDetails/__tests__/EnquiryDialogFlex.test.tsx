import React from 'react'
import EnquiryDialogFlex from '../EnquiryDialogFlex'

import space1 from 'services/space/__mocks__/space1.json'
import MockDate from 'mockdate'

import { render, fireEvent, prettyDOM } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { EnquiryType } from 'services/enquiry'

jest.mock('react-intl', () => ({
  useIntl: () => ({ formatMessage: (s: any) => s.id }),
  createIntl: () => ({ formatMessage: (s: any) => s.id }),
  createIntlCache: jest.fn(),
}))

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useParams: () => ({
    spaceId: '0',
  }),
  Link: () => null,
}))

const mockDispatch = jest.fn()
const mockState: any = {
  negotiation: {
    termsFields: {
      '1': 'Area',
      '2': 'Rent',
      '3': 'Start',
      '4': 'Duration',
      '5': 'Frequency',
      '6': 'AdditionalCosts',
      '7': 'Deposit',
      '8': 'ServiceCharges',
      '9': 'DepositType',
    },
  },
  space: { selectedSpace: { Document: { ...space1 } } },
}
jest.mock('react-redux', () => ({
  ...jest.requireActual('react-redux'),
  useDispatch: () => mockDispatch,
  useSelector: (selector: (state: any) => any) => selector(mockState),
}))
it('renders without crashing', async () => {
  MockDate.set(1434319925275)
  const setOpen = jest.fn()
  const wrapper = render(<EnquiryDialogFlex open={true} setOpen={setOpen} />)
  // Dialog is not in the component, snapshot the whole screen
  expect(prettyDOM()).toMatchSnapshot()
  expect(
    wrapper.queryByText('ENQUIRY_DIALOG_FLEX_PEOPLE'),
  ).not.toBeInTheDocument()
  const hotDesksRadioButton = wrapper.getByDisplayValue(EnquiryType.HotDesk)
  fireEvent(
    hotDesksRadioButton,
    new MouseEvent('click', {
      bubbles: true,
      cancelable: true,
    }),
  )

  expect(wrapper.queryByText('ENQUIRY_DIALOG_FLEX_PEOPLE')).toBeInTheDocument()
  expect(
    wrapper.queryByText('50 DETAILS_SERVICED_OFFICES_PEOPLE - 2 kr / Daily'),
  ).not.toBeInTheDocument()
  const SevicedOfficesRadioButton = wrapper.getByDisplayValue(
    EnquiryType.ServicedOffice,
  )
  fireEvent(
    SevicedOfficesRadioButton,
    new MouseEvent('click', {
      bubbles: true,
      cancelable: true,
    }),
  )
  expect(
    wrapper.queryByText('50 DETAILS_SERVICED_OFFICES_PEOPLE - 2 kr / Daily'),
  ).toBeInTheDocument()
  expect(
    wrapper.queryByText('ENQUIRY_DIALOG_FLEX_PEOPLE'),
  ).not.toBeInTheDocument()

  MockDate.reset()
})
