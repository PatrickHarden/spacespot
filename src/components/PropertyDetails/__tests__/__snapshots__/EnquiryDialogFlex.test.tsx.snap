// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
"[36m<body[39m
  [33mstyle[39m=[32m\\"padding-right: 0px; overflow: hidden;\\"[39m
[36m>[39m
  [36m<div[39m
    [33maria-hidden[39m=[32m\\"true\\"[39m
  [36m/>[39m
  [36m<div[39m
    [33mclass[39m=[32m\\"MuiDialog-root\\"[39m
    [33mrole[39m=[32m\\"presentation\\"[39m
    [33mstyle[39m=[32m\\"position: fixed; z-index: 1300; right: 0px; bottom: 0px; top: 0px; left: 0px;\\"[39m
  [36m>[39m
    [36m<div[39m
      [33maria-hidden[39m=[32m\\"true\\"[39m
      [33mclass[39m=[32m\\"MuiBackdrop-root\\"[39m
      [33mstyle[39m=[32m\\"opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;\\"[39m
    [36m/>[39m
    [36m<div[39m
      [33mdata-test[39m=[32m\\"sentinelStart\\"[39m
      [33mtabindex[39m=[32m\\"0\\"[39m
    [36m/>[39m
    [36m<div[39m
      [33mclass[39m=[32m\\"MuiDialog-container MuiDialog-scrollPaper\\"[39m
      [33mrole[39m=[32m\\"none presentation\\"[39m
      [33mstyle[39m=[32m\\"opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;\\"[39m
      [33mtabindex[39m=[32m\\"-1\\"[39m
    [36m>[39m
      [36m<div[39m
        [33mclass[39m=[32m\\"MuiPaper-root MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm MuiPaper-rounded\\"[39m
        [33mrole[39m=[32m\\"dialog\\"[39m
      [36m>[39m
        [36m<div[39m
          [33mclass[39m=[32m\\"MuiDialogTitle-root sc-dnqmqq eqURtQ\\"[39m
        [36m>[39m
          [36m<h2[39m
            [33mclass[39m=[32m\\"MuiTypography-root MuiTypography-h6\\"[39m
          [36m>[39m
            [0mENQUIRY_DIALOG_TITLE[0m
            [36m<svg[39m
              [33maria-hidden[39m=[32m\\"true\\"[39m
              [33mclass[39m=[32m\\"MuiSvgIcon-root\\"[39m
              [33mfocusable[39m=[32m\\"false\\"[39m
              [33mrole[39m=[32m\\"presentation\\"[39m
              [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
            [36m>[39m
              [36m<path[39m
                [33md[39m=[32m\\"M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z\\"[39m
              [36m/>[39m
            [36m</svg>[39m
          [36m</h2>[39m
        [36m</div>[39m
        [36m<div[39m
          [33mclass[39m=[32m\\"MuiDialogContent-root\\"[39m
        [36m>[39m
          [36m<p>[39m
            [0mENQUIRY_DIALOG_REGISTERED[0m
            [36m<button[39m
              [33mclass[39m=[32m\\"sc-jKJlTe dawH\\"[39m
            [36m>[39m
              [0mENQUIRY_DIALOG_SIGNIN[0m
            [36m</button>[39m
          [36m</p>[39m
          [36m<h2[39m
            [33mclass[39m=[32m\\"sc-gZMcBi drEsAK\\"[39m
          [36m>[39m
            [0mENQUIRY_DIALOG_SUBTITLE_FLEX[0m
          [36m</h2>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-jTzLTM gioBvP\\"[39m
          [36m>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-jzJRlG idnbLP\\"[39m
            [36m>[39m
              [36m<div[39m
                [33mclass[39m=[32m\\"sc-gzVnrw hNBHHv\\"[39m
              [36m>[39m
                [36m<div[39m
                  [33mclass[39m=[32m\\"MuiFormGroup-root\\"[39m
                  [33mrole[39m=[32m\\"radiogroup\\"[39m
                [36m>[39m
                  [36m<label[39m
                    [33mclass[39m=[32m\\"MuiFormControlLabel-root sc-bZQynM nMYUf\\"[39m
                  [36m>[39m
                    [36m<span[39m
                      [33maria-disabled[39m=[32m\\"false\\"[39m
                      [33mclass[39m=[32m\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-100 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary\\"[39m
                    [36m>[39m
                      [36m<span[39m
                        [33mclass[39m=[32m\\"MuiIconButton-label\\"[39m
                      [36m>[39m
                        [36m<input[39m
                          [33mclass[39m=[32m\\"PrivateSwitchBase-input-103\\"[39m
                          [33mname[39m=[32m\\"type\\"[39m
                          [33mtype[39m=[32m\\"radio\\"[39m
                          [33mvalue[39m=[32m\\"HotDesk\\"[39m
                        [36m/>[39m
                        [36m<div[39m
                          [33mclass[39m=[32m\\"PrivateRadioButtonIcon-root-116\\"[39m
                        [36m>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-117\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                        [36m</div>[39m
                      [36m</span>[39m
                    [36m</span>[39m
                    [36m<span[39m
                      [33mclass[39m=[32m\\"MuiTypography-root MuiFormControlLabel-label MuiTypography-body1\\"[39m
                    [36m>[39m
                      [0mENQUIRY_DIALOG_FLEX_HOT_DESKS[0m
                    [36m</span>[39m
                  [36m</label>[39m
                  [36m<label[39m
                    [33mclass[39m=[32m\\"MuiFormControlLabel-root sc-bZQynM nMYUf\\"[39m
                  [36m>[39m
                    [36m<span[39m
                      [33maria-disabled[39m=[32m\\"false\\"[39m
                      [33mclass[39m=[32m\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-100 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary\\"[39m
                    [36m>[39m
                      [36m<span[39m
                        [33mclass[39m=[32m\\"MuiIconButton-label\\"[39m
                      [36m>[39m
                        [36m<input[39m
                          [33mclass[39m=[32m\\"PrivateSwitchBase-input-103\\"[39m
                          [33mname[39m=[32m\\"type\\"[39m
                          [33mtype[39m=[32m\\"radio\\"[39m
                          [33mvalue[39m=[32m\\"FixedDesk\\"[39m
                        [36m/>[39m
                        [36m<div[39m
                          [33mclass[39m=[32m\\"PrivateRadioButtonIcon-root-116\\"[39m
                        [36m>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-117\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                        [36m</div>[39m
                      [36m</span>[39m
                    [36m</span>[39m
                    [36m<span[39m
                      [33mclass[39m=[32m\\"MuiTypography-root MuiFormControlLabel-label MuiTypography-body1\\"[39m
                    [36m>[39m
                      [0mENQUIRY_DIALOG_FLEX_FIXED_DESKS[0m
                    [36m</span>[39m
                  [36m</label>[39m
                  [36m<label[39m
                    [33mclass[39m=[32m\\"MuiFormControlLabel-root sc-bZQynM nMYUf\\"[39m
                  [36m>[39m
                    [36m<span[39m
                      [33maria-disabled[39m=[32m\\"false\\"[39m
                      [33mclass[39m=[32m\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-100 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary\\"[39m
                    [36m>[39m
                      [36m<span[39m
                        [33mclass[39m=[32m\\"MuiIconButton-label\\"[39m
                      [36m>[39m
                        [36m<input[39m
                          [33mclass[39m=[32m\\"PrivateSwitchBase-input-103\\"[39m
                          [33mname[39m=[32m\\"type\\"[39m
                          [33mtype[39m=[32m\\"radio\\"[39m
                          [33mvalue[39m=[32m\\"ServicedOffice\\"[39m
                        [36m/>[39m
                        [36m<div[39m
                          [33mclass[39m=[32m\\"PrivateRadioButtonIcon-root-116\\"[39m
                        [36m>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                          [36m<svg[39m
                            [33maria-hidden[39m=[32m\\"true\\"[39m
                            [33mclass[39m=[32m\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-117\\"[39m
                            [33mfocusable[39m=[32m\\"false\\"[39m
                            [33mrole[39m=[32m\\"presentation\\"[39m
                            [33mviewBox[39m=[32m\\"0 0 24 24\\"[39m
                          [36m>[39m
                            [36m<path[39m
                              [33md[39m=[32m\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"[39m
                            [36m/>[39m
                          [36m</svg>[39m
                        [36m</div>[39m
                      [36m</span>[39m
                    [36m</span>[39m
                    [36m<span[39m
                      [33mclass[39m=[32m\\"MuiTypography-root MuiFormControlLabel-label MuiTypography-body1\\"[39m
                    [36m>[39m
                      [0mENQUIRY_DIALOG_FLEX_SERVICED_OFFICE[0m
                    [36m</span>[39m
                  [36m</label>[39m
                [36m</div>[39m
              [36m</div>[39m
            [36m</div>[39m
          [36m</div>[39m
          [36m<h2[39m
            [33mclass[39m=[32m\\"sc-gZMcBi sc-gqjmRU iuRbBQ\\"[39m
          [36m>[39m
            [0mENQUIRY_DIALOG_ADDITIONAL_INFO[0m
          [36m</h2>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-bdVaJa dlaMlh sc-ckVGcZ fnmRDw\\"[39m
          [36m>[39m
            [36m<input[39m
              [33mclass[39m=[32m\\"sc-bwzfXH jHsaxX\\"[39m
              [33mid[39m=[32m\\"enquiry_your_name\\"[39m
              [33mplaceholder[39m=[32m\\"ENQUIRY_DIALOG_NAME\\"[39m
              [33mtype[39m=[32m\\"text\\"[39m
              [33mvalue[39m=[32m\\"\\"[39m
            [36m/>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-htpNat gctktF\\"[39m
            [36m/>[39m
          [36m</div>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-fjdhpX eblHdy\\"[39m
          [36m>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-bdVaJa dlaMlh sc-ckVGcZ fnmRDw\\"[39m
            [36m>[39m
              [36m<input[39m
                [33mclass[39m=[32m\\"sc-bwzfXH jHsaxX\\"[39m
                [33mid[39m=[32m\\"enquiry_email\\"[39m
                [33mplaceholder[39m=[32m\\"ENQUIRY_DIALOG_EMAIL\\"[39m
                [33mtype[39m=[32m\\"text\\"[39m
                [33mvalue[39m=[32m\\"\\"[39m
              [36m/>[39m
              [36m<div[39m
                [33mclass[39m=[32m\\"sc-htpNat gctktF\\"[39m
              [36m/>[39m
            [36m</div>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-bdVaJa dlaMlh sc-ckVGcZ fnmRDw\\"[39m
            [36m>[39m
              [36m<input[39m
                [33mclass[39m=[32m\\"sc-bwzfXH jHsaxX\\"[39m
                [33mid[39m=[32m\\"enquiry_email2\\"[39m
                [33mplaceholder[39m=[32m\\"ENQUIRY_DIALOG_EMAIL2\\"[39m
                [33mtype[39m=[32m\\"text\\"[39m
                [33mvalue[39m=[32m\\"\\"[39m
              [36m/>[39m
              [36m<div[39m
                [33mclass[39m=[32m\\"sc-htpNat gctktF\\"[39m
              [36m/>[39m
            [36m</div>[39m
          [36m</div>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-bdVaJa dlaMlh sc-ckVGcZ fnmRDw\\"[39m
          [36m>[39m
            [36m<input[39m
              [33mclass[39m=[32m\\"sc-bwzfXH jHsaxX\\"[39m
              [33mid[39m=[32m\\"enquiry_company_name\\"[39m
              [33mplaceholder[39m=[32m\\"ENQUIRY_DIALOG_COMPANY\\"[39m
              [33mtype[39m=[32m\\"text\\"[39m
              [33mvalue[39m=[32m\\"\\"[39m
            [36m/>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-htpNat gctktF\\"[39m
            [36m/>[39m
          [36m</div>[39m
          [36m<label[39m
            [33mclass[39m=[32m\\"sc-dxgOiQ gfVpUG\\"[39m
          [36m>[39m
            [0m [0m
            [0mENQUIRY_DIALOG_PASSWORD[0m
          [36m</label>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-bdVaJa dlaMlh sc-ckVGcZ fnmRDw\\"[39m
          [36m>[39m
            [36m<input[39m
              [33mclass[39m=[32m\\"sc-bwzfXH jHsaxX\\"[39m
              [33mid[39m=[32m\\"enquiry_password\\"[39m
              [33mplaceholder[39m=[32m\\"password\\"[39m
              [33mtype[39m=[32m\\"password\\"[39m
              [33mvalue[39m=[32m\\"\\"[39m
            [36m/>[39m
            [36m<div[39m
              [33mclass[39m=[32m\\"sc-htpNat gctktF\\"[39m
            [36m/>[39m
          [36m</div>[39m
          [36m<label[39m
            [33mclass[39m=[32m\\"sc-dxgOiQ gfVpUG\\"[39m
          [36m>[39m
            [0m [0m
            [0mENQUIRY_DIALOG_MESSAGE[0m
          [36m</label>[39m
          [0m [0m
          [36m<textarea[39m
            [33mclass[39m=[32m\\"sc-VigVT fzKNXL\\"[39m
          [36m>[39m
            [0mENQUIRY_DIALOG_DEAFULT_MESSAGE[0m
          [36m</textarea>[39m
        [36m</div>[39m
        [36m<div[39m
          [33mclass[39m=[32m\\"MuiDialogActions-root MuiDialogActions-spacing\\"[39m
        [36m>[39m
          [36m<p[39m
            [33mclass[39m=[32m\\"sc-eNQAEJ hVnkXH\\"[39m
          [36m>[39m
            [0mENQUIRY_DIALOG_NOTICE[0m
            [0m [0m
            [0m [0m
            [0mENQUIRY_DIALOG_NOTICE_AND[0m
            [0m [0m
          [36m</p>[39m
          [36m<div[39m
            [33mclass[39m=[32m\\"sc-htoDjs ioLkuh\\"[39m
          [36m>[39m
            [36m<button[39m
              [33mclass[39m=[32m\\"sc-EHOje sc-iwsKbI gspjxH\\"[39m
              [33mcolor[39m=[32m\\"primary\\"[39m
              [33mdata-auto[39m=[32m\\"submit-enquiry-dialog-flex\\"[39m
              [33mid[39m=[32m\\"dialogOK\\"[39m
            [36m>[39m
              [0mENQUIRY_DIALOG_BUTTON[0m
            [36m</button>[39m
          [36m</div>[39m
        [36m</div>[39m
      [36m</div>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mdata-test[39m=[32m\\"sentinelEnd\\"[39m
      [33mtabindex[39m=[32m\\"0\\"[39m
    [36m/>[39m
  [36m</div>[39m
[36m</body>[39m"
`;
